requirements.txt

```diff
@@ -0,0 +1,24 @@
+adium-theme-ubuntu==0.3.4
```

Comment:
Make sure you run `pip freeze > requirements.txt` in the terminal where you have enabled your virtual environment

Reply:
Your build is failing due to the presence of weird requirements